terms.adoc

```diff
@@ -0,0 +1,130 @@
+////
+Apply pseudo legal capitalization to the main document:
+cat eclipse_development_process.adoc | sed -E -e "`grep -P -oh "^.+(?=\s+\:\:)" terms.adoc | while read -r term; do echo -ne "s|$term|$term|gI;"; done`"
+////
+
+[glossary]
+
+Architecture Council ::
+The Eclipse Architecture Council (AC) is a council of experienced Eclipse committers responsible for identifying and tackling any issues that hinder continued technological success and innovation, widespread adoption, and future growth of Eclipse technology.
+
+Board of Directors ::
+The business and technical affairs of the Eclipse Foundation are managed by or under the direction of the Eclipse Board of Directors. 
+
+Check Point Review ::
+Check Point Review is a category of Review that includes: Creation Review, Progress Review, Release Review, and Graduation Review.
+
+Committer ::
+A committer is a software developer who has the necessary rights to make decisions regarding a Project.
+
+Contributor ::
+A Contributor is anybody who makes contributions to the project.
+
+Creation Review ::
+A Creation Review is a type of Review that marks a Project's transition from the Proposal Phase to the Incubation Phase.
+
+Developer ::
+A Committer or Contributor.
+
+Ecosystem ::
+A commercial Ecosystem is a system in which companies, organizations, and individuals all work together for mutual benefit.
```

Comment:
Is this really "commercial" then? This would seem to exclude some research activities or non-profit organizations. Isn't it rather a "collaborative" or "technical" ecosystem instad?

Reply:
Yes, I agree. No need to emphasize "commercial".

Reply:
I cribbed that from somewhere, but can't remember where... Regardless, I agree.